tests/ui/UnreadIndicatorsTest.js

```diff
@@ -0,0 +1,395 @@
+/* eslint-disable @lwc/lwc/no-async-await */
+import React from 'react';
+import Onyx from 'react-native-onyx';
+import {Linking, AppState} from 'react-native';
+import {fireEvent, render, act} from '@testing-library/react-native';
+import lodashGet from 'lodash/get';
+import moment from 'moment';
+import App from '../../src/App';
+import CONST from '../../src/CONST';
+import ONYXKEYS from '../../src/ONYXKEYS';
+import waitForPromisesToResolve from '../utils/waitForPromisesToResolve';
+import * as TestHelper from '../utils/TestHelper';
+import appSetup from '../../src/setup';
+import fontWeightBold from '../../src/styles/fontWeight/bold';
+import * as AppActions from '../../src/libs/actions/App';
+import ReportHeaderViewBackButton from '../../src/pages/home/ReportHeaderViewBackButton';
+import ReportActionsView from '../../src/pages/home/report/ReportActionsView';
+import * as NumberUtils from '../../src/libs/NumberUtils';
+import LocalNotification from '../../src/libs/Notification/LocalNotification';
+import * as Report from '../../src/libs/actions/Report';
+
+beforeAll(() => {
+    global.fetch = TestHelper.getGlobalFetchMock();
```

Comment:
Please add a comment to explain why this is being mocked. Should this go with all our other mocks in the setup file?

Reply:
Created a follow up for this here: https://github.com/Expensify/Expensify/issues/228440